CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to IronCalc
```

Comment:
I think this file is a left over from the previous PR and it is already merged into master